code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -78,7 +78,8 @@
 	SEND_SIGNAL(I, COMSIG_ITEM_ATTACK_ZONE, src, user, affecting)
 	send_item_attack_message(I, user, parse_zone(affecting.body_zone))
 	if(I.force)
-		apply_damage(I.force, I.damtype, affecting)
+		var/armour_block = run_armor_check(affecting, MELEE, armour_penetration = I.armour_penetration)
+		apply_damage(I.force, I.damtype, affecting, armour_block)
```

Comment:
You should move the armor (again typo ‼️ ) into the apply damage proc and pass more args to it.

Reply:
Not all uses of apply_damage use armour. This would be a major rewrite to damage application and is out of scope for this PR.